back/backend/handlers/admin/show_users.py

```diff
@@ -0,0 +1,89 @@
+from json import loads
+from flask import Blueprint, jsonify, request
+from google.cloud import ndb
+
+from backend.models.wca.person import Person
+from backend.lib import auth
+from backend.models.user import User, Roles
+
+bp = Blueprint('show_users', __name__)
+client = ndb.Client()
+
+
+@bp.route('/get_users')
+def get_users():
+    with client.context():
+        me = auth.user()
+        if not me or not me.HasAnyRole(Roles.AdminRoles()):
+            return jsonify({"error": "Forbidden"}), 403
```

Comment:
`/update_provinces` just did `abort(403)`. I have no preference between the two, but let's make the application consistent please.